### PR TITLE
Update process.square.js to use the WebPayments SDK instead of SqPaymentForm

### PR DESCRIPTION
**Pull request by CupNoodles**
Deprecation warning no longer shows up in the web console. 

I've only tested in sandbox, but can confirm that behaviors are the same between the old SqPaymentForms implementation and this one. I've 'tested' the SCA process in sandbox with the values described in https://developer.squareup.com/docs/testing/test-values, but as I don't have an EU bank account I have no clue what this looks like in production.

### CODE REVIEW

File: assets/process.square.js [hunk 1]
```diff
@@ -5,36 +5,51 @@
...
+        });
 
         this.$checkoutForm.on('submitCheckoutForm', $.proxy(this.submitFormHandler, this))
     }
 
-    ProcessSquare.prototype.submitFormHandler = function (event) {
+    ProcessSquare.prototype.initializeCard = async function(payments) {
+
+        // Customize the CSS for WebPayments SDK elements
+        this.card = await payments.card({
```
Comment (sampoyigi) [inline assets/process.square.js:34]: Move this has to ProcessSquare.DEFAULTS

**Review by sampoyigi — CHANGES_REQUESTED** [review]

**Comment by sampoyigi** [comment]
Apart from the requested changes, everything else appears to be in order, and I'll come back later to further check and test it. Thank you for your time and work!

**Comment by CupNoodles** [comment]
Moved the card styles over! Should I be trying to fix these CI failures?

**Comment by sampoyigi** [comment]
Yup

**Comment by sampoyigi** [comment]
Thank you for your efforts; apart from a typo with the verificationToken, I was able to test without issue.

**Pull request merged**